Modules/IO/NIFTI/test/itkNiftiReadWriteDirectionTest.cxx

```diff
@@ -107,6 +109,7 @@ itkNiftiReadWriteDirectionTest(int ac, char * av[])
   }
   if (!is_close_qform_converted)
   {
+    std::cerr << "Error: sform reconstructed from qform is not similar to qform" << std::endl;
     std::cerr << LPSLabel_nosform_direction << " \n\n" << reread_direction << std::endl;
```

Comment:
Maybe this last print message could be improved to report what each variable corresponds to. Something like, e.g.
```
std::cerr << "Original qform-only image direction: " << LPSLabel_nosform_direction << std::endl;
std::cerr << "Original qform-only image re-written as qfrom and sform-complete image direction:" << reread_direction << std::endl;
```

Reply:
Addressed in 7f16366.